### ISSUE TITLE
from collections import defaultdict

### ISSUE THREAD

**Issue by arani-mohammad** [IBMDecisionOptimization/docplex-examples#16]
Hi all,
When I have installed CPLEX Studio12.9 I have noticed in the installation directory, there is a folder named "example". Since I am programming with python I was interested in the relevant session. 
I have two questions:
One is what is the following library which is imported and used in some example files:
`from collections import defaultdict`
Is this [LINK ](https://pypi.org/project/collections2/) refers to the above library?
Two is that I know CPLEX provides tools for multi-objective problems. Now is there an easy example for the sake of learning and introducing the tools and parameters?
The following link is where I referred but could not digest most of it:
[LINK](https://www.ibm.com/support/knowledgecenter/SSSA5P_12.9.0/ilog.odms.cplex.help/CPLEX/UsrMan/topics/multiobj/partMultiobj_title_synopsis.html)
Basically, is there a small example showing how to use the following commands (_provided that they are the correct ones_) and set the parameters of multi-objective problems' [Parameters](https://www.ibm.com/support/knowledgecenter/en/SSSA5P_12.9.0/ilog.odms.cplex.help/CPLEX/UsrMan/topics/multiobj/multiobj_specifying.html)
`mdl.set_multi_objective()`
`mdl.set_multi_objective_exprs()`
Thanks

**Comment by vlkong** [comment]
Hi,

Those questions are not docplex related. The cplex team is monitoring questions on https://stackoverflow.com/questions/tagged/cplex so you should ask there for questions about the CPLEX examples.

However, here's some answers:
1 - collections is the collections package from python. See https://docs.python.org/3/library/collections.html

2 - Please have a look at the diet.py examples (in cplex/examples/src/python). It demonstrates how to set a second objective,

**Comment by arani-mohammad** [comment]
I really appreciated your time.
